Source/UI/NewUI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -70,7 +71,23 @@ final class PaymentViewController: UIViewController {
     return view
   }()
 
-  private let headerBackgroundView = UIView().disabledAutoresizingIntoConstraints()
+  private lazy var payButtonView: ButtonView = {
+    let view = ButtonView()
+    view.delegate = self
+    view.isEnabled = false
+    return view
+  }()
+
+  private lazy var activityIndicator: UIActivityIndicatorView = {
+    let view =  UIActivityIndicatorView(frame: CGRect(x: 0, y: 0, width: 20, height: 20))
+    let barButton = UIBarButtonItem(customView: view)
+    navigationItem.setRightBarButton(barButton, animated: true)
+    return view
+  }()
```

Comment:
This is useful for some dev feedback, but realistically a small loading indicator in navigation bar is not serving any other purpose than dev checks. And we may end up forgetting it, so might be useful to delete and make sure @patrick-hoban-cko has some task for some type of loading state on the screen.

Reply:
We are waiting for Alexander the designer about this. for now, it is temp solution.

Reply:
Sure, its a good temp solution for you to test behaviour, but not a good solution to merge into a main develop branch. That will absolutely not stay like this and you only risk adding code that will be forgotten on refactor, please remove.